testapps/ClassLibraryWithPrecompiledViews/project.json

```diff
@@ -0,0 +1,37 @@
+﻿{
+  "version": "1.0.0-*",
+  "buildOptions": {
+    "preserveCompilationContext": true,
+    "emitEntryPoint": true
+  },
+  "packOptions": {
+    "files": {
+      "mappings": {
+        "lib/": "obj/precompiled/" 
+      }
+    }
+  },
+  "dependencies": {
+    "Microsoft.AspNetCore.Mvc": "1.1.0-*",
+    "Microsoft.AspNetCore.Mvc.Razor.Precompilation.Design": {
+      "version": "1.0.0-*",
+      "target": "package",
+      "type": "build"
+    }
+  },
+  "tools": {
+    "Microsoft.AspNetCore.Mvc.Razor.Precompilation.Tools": "1.0.0-*"
+  },
+
+  "frameworks": {
+    "netcoreapp1.0": {
```

Comment:
The intent was to show what it looks like to produce packages containing controllers and precompiled views today. It's a scenario people keep trying to do and it's nice to have some level of test coverage to make sure it keeps working.